packages/shared/src/aws/tile.metadata.tileset.ts

```diff
@@ -137,6 +140,14 @@ export class TileMetadataTileSet extends TaggedTileMetadata<TileMetadataSetRecor
         return `ts_${record.name}_${record.projection}_${tag}`;
     }
 
+    idSplit(record: TileMetadataSetRecord): { name: string; projection: string; tag: string } | null {
```

Comment:
I dont think we should leave projection as `string` we should use `projection: Epsg` or `projection: EpsgCode` which is the result of the  `Epsg.parse`